gui/screens/settings.py

```diff
@@ -268,6 +268,16 @@ def __init__(self):
         super(EditAccountOrderSettings, self).__init__()
         self._account_item = QObject('settingsContentBaseScrollView_draggableDelegate_StatusDraggableListItem')
         self._accounts_list = QObject('statusDesktop_mainWindow')
+        self._text_on_top = QObject('settingsContentBaseScrollView_StatusBaseText')
+        self._back_button = Button('main_toolBar_back_button')
+
+    @property
+    @allure.step('Get text labels')
+    def text_labels(self):
```

Comment:
Rename it to understand which labels you collect here
All labels from the settings view?
When I open it how can I use it?

Reply:
I collect here all text labels from Edit order account settings. I renamed how I see it. Is that ok now?